src/source/Ice/IceAgent.h

```diff
@@ -80,24 +80,23 @@ typedef struct __IceAgent* PIceAgent;
  * Internal structure tracking ICE server parameters for diagnostics and metrics/stats
  */
 typedef struct {
-    CHAR url[MAX_STATS_STRING_LENGTH + 1];      //!< STUN/TURN server URL
-    CHAR protocol[MAX_STATS_STRING_LENGTH + 1]; //!< Valid values: UDP, TCP
-    INT32 port;                                 //!< Port number used by client
-    UINT64 totalRequestsSent;                   //!< Total amount of requests that have been sent to the server
-    UINT64 totalResponsesReceived;              //!< Total number of responses received from the server
-    UINT64 totalRoundTripTime;                  //!< Sum of RTTs of all the requests for which response has been received
+    CHAR url[MAX_STATS_STRING_LENGTH + 1];  //!< STUN/TURN server URL
+    CHAR protocol[MAX_PROTOCOL_LENGTH + 1]; //!< Valid values: UDP, TCP
+    INT32 port;                             //!< Port number used by client
+    UINT64 totalRequestsSent;               //!< Total amount of requests that have been sent to the server
+    UINT64 totalResponsesReceived;          //!< Total number of responses received from the server
+    UINT64 totalRoundTripTime;              //!< Sum of RTTs of all the requests for which response has been received
 } RtcIceServerDiagnostics, *PRtcIceServerDiagnostics;
 
 typedef struct {
-    DOMString url; //!< For local candidates this is the URL of the ICE server from which the candidate was obtained
-    DOMString transportId[MAX_STATS_STRING_LENGTH + 1]; //!< ID of object that was inspected for RTCTransportStats
-    CHAR address[KVS_IP_ADDRESS_STRING_BUFFER_LEN];     //!< IPv4 or IPv6 address of the candidate
-    DOMString protocol;                                 //!< Valid values: UDP, TCP
-    DOMString relayProtocol;                            //!< Protocol used by endpoint to communicate with TURN server.
-                                                        //!< Valid values: UDP, TCP, TLS
-    INT32 priority;                                     //!< Computed using the formula in https://tools.ietf.org/html/rfc5245#section-15.1
-    INT32 port;                                         //!< Port number of the candidate
-    DOMString candidateType;                            //!< Type of local/remote ICE candidate
+    DOMString url;                                  //!< For local candidates this is the URL of the ICE server from which the candidate was obtained
+    CHAR address[KVS_IP_ADDRESS_STRING_BUFFER_LEN]; //!< IPv4 or IPv6 address of the candidate
```

Comment:
I saw `IP_ADDR_STR_LENGTH + 1` in another part of this PR, is this storing something different? The comment is the same as in `RtcIceCandidateStats`.

Since these strings use a lot of the same types, suggest to add and use typedefs for these char arrays with optimized lengths.

Reply:
Neither `IP_ADDR_STR_LENGTH` nor `KVS_IP_ADDRESS_STRING_BUFFER_LEN` were modified or used differently in this PR. Will leave this comment unresolved for visibility for future improvements.